apps/giddh-cordova/giddh/config.xml

```diff
@@ -1,176 +1,153 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-packageName="com.app.Giddh" ios-CFBundleIdentifier="com.app.giddh" version="2.3.2"
-        xmlns="http://www.w3.org/ns/widgets"
-        xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget android-packageName="com.app.Giddh" ios-CFBundleIdentifier="com.app.giddh" version="2.3.2" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
     <name>Giddh</name>
     <description>
         A sample Apache Cordova application that responds to the deviceready event.
     </description>
     <author email="govi2010@gmail.com" href="http://giddh.com">
```

Comment:
Replace email by shubhendra sir's email and change url to https://giddh.com

Reply:
Done